src/hb-ot-glyf-table.hh

```diff
@@ -280,16 +282,18 @@ struct glyf
 						 composite);
     }
 
+    enum simple_glyph_flag_t {
+      FLAG_X_SHORT = 0x02,
+      FLAG_Y_SHORT = 0x04,
+      FLAG_REPEAT = 0x08,
+      FLAG_X_SAME = 0x10,
+      FLAG_Y_SAME = 0x20
+    };
```

Comment:
This was needed on llvm-gcc-4.2 exported symbols tests, without it was saying these are exported.